web-components/src/components/combobox/ComboBox.ts

```diff
@@ -819,7 +820,7 @@ export namespace ComboBox {
                 type="text"
                 role="combobox"
                 aria-autocomplete="both"
-                aria-label="input"
+                aria-label="Combobox Input"
```

Comment:
why hard coded ariaLabel? it should be passed from application. so that it can be localized